src/Compilers/Core/Portable/xlf/CodeAnalysisResources.de.xlf

```diff
@@ -678,12 +678,12 @@
       </trans-unit>
       <trans-unit id="KeepAliveWithoutShared">
         <source>'/keepalive' option is only valid with '/shared' option.</source>
-        <target state="translated">Die /keepalive-Option ist nur zusammen mit der /shared-Option gültig.</target>
+        <target state="translated">'Die "/keepalive"-Option ist nur zusammen mit der "/shared"-Option gültig.</target>
         <note />
       </trans-unit>
       <trans-unit id="MismatchedVersion">
         <source>Roslyn compiler server reports different protocol version than build task.</source>
-        <target state="translated">Der Roslyn-Compilerserver meldet eine andere Protokollversion als der Buildtask.</target>
+        <target state="translated">Roslyn-Compilerserver meldet unterschiedliche Protokollversion als in Erstellungsaufgabe.</target>
```

Comment:
"meldet *eine andere* Protokollversion als *die* Erstellungsaufgabe" would be a better translation I think.